templates/document_categories/general_list.html

```diff
@@ -14,10 +14,10 @@
 </style>
 {/literal}
 <script type="text/javascript">
-var deleteLabel="{xl t="Delete"}";
+var deleteLabel="{xl t='Delete'}";
 </script>
 
-<script type="text/javascript" src="../../library/js/DocumentTreeMenu.js"></script>
+<script type="text/javascript" src="library/js/DocumentTreeMenu.js"></script>
```

Comment:
Not sure where this template is called from and how that affects the src...but if globals.php is called with it, then we have a path variable for the document root that can be passed to javascript.  I am thinking we need to pass that to js globals in any case.
If this works...then fine.  We should still review the globals and session variables some time in the future with JS in mind.
Good catch on the quotes.